cnd/migrations/2020-05-06-105000_created-swap-tables/up.sql

```diff
@@ -0,0 +1,64 @@
+-- Your SQL goes here
+
+CREATE TABLE swaps
+(
+    id INTEGER			NOT NULL PRIMARY KEY,
+    local_swap_id UNIQUE	NOT NULL,
+    role			NOT NULL,
+    counterparty_peer_id	NOT NULL
+);
+
+CREATE TABLE secret_hashes
+(
+    id INTEGER			NOT NULL PRIMARY KEY,
+    swap_id INTEGER		NOT NULL,
```

Comment:
Shouldn't need this if we have the foreign key.

Reply:
This _is_ the foreign key

Reply:
Same goes for all comments below.  We use the `swap_id` in each table as the foreign key.  cnd swap ids are referred to as `local_swap_id` and `shared_swap_id` throughout the schema.

Reply:
This is inline with how diesel expects things to be named, the type (INTEGER) is also in line with this usage.

Reply:
Ah okay, my mistake, I thought this was the UUID :)